examples/sample-app/application-template-custombuild.json

```diff
@@ -66,6 +66,7 @@
       "parameters": {
         "output": {
           "to": {
+            "kind": "ImageStream",
```

Comment:
yeah, in both cases I think we agreed that you don't have to specify the Kind and we assume that the Name is the 'default' Kind. From and To has different defaults, right? @bparees